docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -34,14 +34,12 @@ All tasks take the same arguments. For example:
 .. code-block:: text
 
    $ uw chgres_cube run --help
-   usage: uw chgres_cube run --config-file PATH --cycle CYCLE [-h] [--version] [--batch] [--dry-run]
-                             [--graph-file PATH] [--quiet] [--verbose]
+   usage: uw chgres_cube run --cycle CYCLE [-h] [--version] [--config-file PATH] [--batch]
+                             [--dry-run] [--graph-file PATH] [--quiet] [--verbose]
 
    A run
 
    Required arguments:
-     --config-file PATH, -c PATH
-         Path to UW YAML config file
```

Comment:
I re-ran many of the CLI invocation examples and update the docs with their current output.